### ISSUE TITLE
Setting animation to false breaks line and scatter error bars

### ISSUE THREAD

**Issue by mossymaker** [sgratzl/chartjs-chart-error-bars#24]
When I set `animation: false` on line and scatter charts, error bars aren't drawn.

**To Reproduce**

1. Visit the CodePen: https://codepen.io/mossymaker/pen/zYRmwPP2. 
1. Toggle the `animation` option.

**Expected behavior**

Error bars render without animation.

**Context**

- Version: chartjs@3, chartjs-chart-error-bars@3
- Browser: Safari 15.5; also happens on Chrome

**Additional context**

**Comment by andrewaarestad** [comment]
I came across this issue due to using `chartjs-node-canvas` which sets animation to false automatically.  Scatter chart error bars aren't showing up, but bar chart error bars work.

Stepping through the code, it looks like the issue is that for scatter and line plots, chart.js has a "directUpdate" mode that gets used when animation is disabled.  In this mode, it does not call `updateElement()` on each element, which results in the custom properties e.g. `xMin, xMax` not getting transferred from the parsed data objects to the `PointWithErrorBar` objects.

If I manually disable `directUpdate` using a step debugger, I was able to get a chart to render with error bars.  However I can't figure out a way to prevent `directUpdate` from being set by changing any configuration.  Even if I alter the `chartjs-node-canvas` library to set `animation: true`, chart.js will override it, since it detects you aren't in a DOM platform.

So I'm not sure if this can be addressed by a change to the `chartjs-chart-error-bars` module.  I might try forking chart.js and adding an option to disable `directUpdate`, I think that would probably work.
